### PR TITLE
Add support for fetching css values of an element

### PR DESCRIPTION
**Pull request by awikman**
Implementing the `GET  /session/{session id}/element/{element id}/css/{property name}` method.

I needed this to fetch the background-color of an element, so an example use case would be something like:
```
  let sel_body = r#"body"#;
  let mut el_body = client.find(Locator::Css(sel_body)).await?;
  let body_background_color = el_body.css("background-color").await?;
```

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -987,6 +987,16 @@ impl Element {
             .collect())
     }
 
+    /// Look up a CSS value for this element by name (color, width, etc)
+    pub async fn css(&mut self, prop: &str) -> Result<Option<String>, error::CmdError> {
+        let cmd = WebDriverCommand::GetCSSValue(self.element.clone(), prop.to_string());
+        match self.client.issue(cmd).await? {
+            Json::String(v) => Ok(Some(v)),
+            Json::Null => Ok(None),
```
Comment (jonhoo) [inline src/lib.rs:995]: What exactly does `None` mean here? Does it mean that the element did not exist, that the attribute was not set, or something else? Would be good to add that to the method documentation!

**Comment by jonhoo** [comment]
This looks good with just a little more documentation! If you could also add a simple test, that'd be great :)

File: src/lib.rs (hunk 1 above)
Comment (awikman) [reply src/lib.rs]: Looks like the css command always returns a string, even if the property is not defined. Must have been some issue with my testing previously.... 14a90c8

**Comment by awikman** [comment]
I'll try to add more as soon as I have time.
Looks like the tests are currently failing. Is there PR/Issue about that already?

File: src/lib.rs [hunk 2]
```diff
@@ -987,6 +987,17 @@ impl Element {
             .collect())
     }
 
+    /// Look up a CSS value for this element by name (color, width, etc)
+    ///
+    /// See <https://www.w3.org/TR/webdriver/#dfn-get-element-css-value>.
+    pub async fn css(&mut self, prop: &str) -> Result<Option<String>, error::CmdError> {
```
Comment (jonhoo) [inline src/lib.rs:993]: Is there still a reason for the method to return `Option<String>`, or should that just be `String` now?

As for documentation, I'd like to see it explicitly document what the caller should expect to get back if the property is malformed or hasn't been set.

**Comment by jonhoo** [comment]
This is now present in [0.19.0-rc.3](https://docs.rs/fantoccini/0.19.0-rc.3/fantoccini/elements/struct.Element.html#method.css_value) and onwards :)

**Pull request closed without merging**